### PR TITLE
Fix HTML linter

### PR DESCRIPTION
**Pull request by Leoism**
+ Fix the HTML linter because it was failing to run the `htmllint`
+ Update the .htmllintrc to be more concise with the [Google Style guidelines](https://google.github.io/styleguide/htmlcssguide.html) for HTML. The most notable ones being using 2 spaces instead of 4 and requiring that id and class names use dashes instead of underscores.

### CODE REVIEW

**Review by dchin2 — APPROVED** [review]
Nice

File: .htmllintrc [hunk 1]
```diff
@@ -17,8 +17,8 @@
         "width"
     ],
     "indent-delta": false,
-    "indent-style": "nonmixed",
-    "indent-width": 4,
+    "indent-style": "spaces",
+    "indent-width": 2,
```
Comment (Leoism) [inline .htmllintrc:21]: https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: .htmllintrc [hunk 2]
```diff
@@ -30,8 +30,8 @@
     "tag-close": true,
     "tag-name-lowercase": true,
     "tag-name-match": true,
-    "tag-self-close": false,
-    "doctype-first": false,
+    "tag-self-close": "never",
+    "doctype-first": true,
```
Comment (Leoism) [inline .htmllintrc:34]: https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: .htmllintrc [hunk 3]
```diff
@@ -44,7 +44,7 @@
     "attr-validate": true,
     "id-no-dup": true,
     "id-class-no-ad": true,
-    "id-class-style": "underscore",
+    "id-class-style": "dash",
```
Comment (Leoism) [inline .htmllintrc:47]: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

**Pull request merged**